src/console/Program.cs

```diff
@@ -115,6 +103,26 @@ public static EventProcessorClient GetEventProcessorClient(BlobContainerClient b
             return eventHubReader;
         }
 
+        public static EventProcessorClientFactoryOptions GetEventProcessorFactoryOptions(IConfiguration config)
```

Comment:
Will we be building these options differently for Paas? I ask because EventHub names in Paas will be prefixed by the Iotconnector name as they all share the same Event Hub Namespace.

Reply:
The names of the variables like "eventHubName" were a bit misleading since it not really an event hub name but rather the type of application we are running so I updated the variable to "applicationType".

Right now we have one code base and two "apps" and depending on whether we are a "devicedata" app or a "normazlieddata" app we are injecting the appropriate startup classes. The name of the actual event hubs doesn't matter as those get passed in as separate environment variables. The app type is an environment variable which I've updated to Console:ApplicationType.